### PR TITLE
LW-161 Replace CSS "fixed" position with React-based positioning

### PR DESCRIPTION
**Pull request by danthewolfe**
CSS alone can’t get this to behave the way we want.

### CODE REVIEW

File: src/components/Contentful/Page/sideNavPresenter.js [hunk 1]
```diff
@@ -1,24 +1,75 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+const SCROLLED = true
+const NOTSCROLLED = false
+const MAGICNUMBER = 245
```
Comment (hbeachey) [inline src/components/Contentful/Page/sideNavPresenter.js]: Solid. If you call it a magic number is it a magic number? 🤔

File: src/components/Contentful/Page/sideNavPresenter.js [hunk 2]
```diff
@@ -1,24 +1,75 @@
...
+    if (window.pageYOffset <= MAGICNUMBER && this.state.positionType === SCROLLED) {
+      this.setState({ positionType: NOTSCROLLED })
+    }
+  }
+
+  scrolledStyle () {
+    if (this.state.positionType === SCROLLED) {
+      return {
+        position: 'fixed',
+        top: '10px',
```
Comment (hbeachey) [inline src/components/Contentful/Page/sideNavPresenter.js]: If the search bar is down and advanced search open this overlaps quite a bit.

File: src/components/Contentful/Page/sideNavPresenter.js (hunk 1 above)
Comment (danthewolfe) [reply src/components/Contentful/Page/sideNavPresenter.js]: :trollface:

File: src/components/Contentful/Page/sideNavPresenter.js (hunk 2 above)
Comment (danthewolfe) [reply src/components/Contentful/Page/sideNavPresenter.js]: This should be fixed now.

**Pull request merged**